actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -221,6 +221,8 @@ def sent?;      synchronize { @sent };      end
 
     # Sets the HTTP status code.
     def status=(status)
+      raise ArgumentError, "#{status} is not a valid response code identifier. See Rack::Utils::SYMBOL_TO_STATUS_CODE "\
```

Comment:
The HTTP status symbols that Rails accepts are documented here: http://guides.rubyonrails.org/layouts_and_rendering.html#the-status-option

I think we should point the user to that section of the guides, instead of to the constant in the Rack library.